Mono.Addins.Gui/Mono.Addins.Gui/ProgressDialog.cs

```diff
@@ -34,9 +34,12 @@ internal partial class ProgressDialog : Gtk.Dialog, IProgressStatus
 		bool cancelled;
 		bool hadError;
 		
+		readonly Gtk.Window parent;
```

Comment:
I think this should be a `WeakReference<Gtk.Window>`

Reply:
Fixed in https://github.com/mono/mono-addins/pull/147/commits/cb5412a783c9d869a91026acfdf57622039cb8bf